app/src/main/java/org/mifos/mobile/ui/fragments/RegistrationFragment.kt

```diff
@@ -229,4 +232,3 @@ class RegistrationFragment : BaseFragment(), RegistrationView {
             return RegistrationFragment()
         }
     }
```

Comment:
@Prashant830 I think you are missing closing curly brace at end that cause gradle build fail. Please run `.\gradlew check  --scan` before commit. Also with this error you might not have successfully build project in your android studio it should have shown you error. Please confirm errors before commiting them. Also there are commits of different error in this pr. Make different branches for different pr. You can find help for git and github on internet.

Reply:
@PatelVatsalB21 you have a code for deleting  specific PR ??

Reply:
In git..

Reply:
@Prashant830   use `git reset hash` (here you need to place hash in numbers of commit previous to yours ) to go back to last merged commit and that will revert your commit locally then you remove unwanted changes and commit back then use `git push -f`. I would suggest to go deep on youtube as explaining here would be difficult.

Reply:
@PatelVatsalB21 thanks bro..

Reply:
hello brother you have a LinkedIn account ?? If yes then  give me your account link.. I want to discuss with  you one my idea..

Reply:
you can connect via [This](https://www.linkedin.com/in/vatsal-patel-919691193) :)